roles/tpa_single_node/vars/main.yml

```diff
@@ -4,14 +4,15 @@ tpa_single_node_base_hostname: trustification
 tpa_single_node_certificates_dir: certs/
 tpa_single_node_config_dir: /etc/rhtpa
 tpa_single_node_kube_manifest_dir: "{{ tpa_single_node_config_dir }}/manifests"
+tpa_single_node_secrets_dir: "{{ tpa_single_node_config_dir }}/secrets"
 tpa_single_node_namespace: trustification
 tpa_single_node_podman_network: tcnet
 tpa_single_node_skip_os_install: true
 tpa_single_node_systemd_directory: /etc/systemd/system
 tpa_single_node_default_empty: ""
 
 # DB_SERVICE
-tpa_single_node_pg_host: 192.168.121.60
+# tpa_single_node_pg_host: 192.168.121.60
```

Comment:
We need to have the db host key, you can leave empty but not commenting

Reply:
@desmax74, the change @i386x made in main.yml  using the ansible set_fact replaces this line.
Meanwhile @i386x, `"{{ ansible_default_ipv4.address }}"` gets the first IP address.
Since the host is setup with several networks how to make sure we capture the right interface ?
```
[root@trustification ~]# ip a
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
    link/ether 52:54:00:b3:9b:7e brd ff:ff:ff:ff:ff:ff
    altname enp0s5
    altname ens5
    inet 192.168.121.3/24 brd 192.168.121.255 scope global dynamic noprefixroute eth0
       valid_lft 2864sec preferred_lft 2864sec
3: eth1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
    link/ether 52:54:00:c3:2d:bf brd ff:ff:ff:ff:ff:ff
    altname enp0s6
    altname ens6
    inet 192.168.121.60/24 brd 192.168.121.255 scope global noprefixroute eth1
       valid_lft forever preferred_lft forever
    inet6 fe80::5054:ff:fec3:2dbf/64 scope link 
       valid_lft forever preferred_lft forever
4: podman1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    link/ether 06:83:03:e7:eb:d2 brd ff:ff:ff:ff:ff:ff
    inet 10.89.0.1/24 brd 10.89.0.255 scope global podman1
       valid_lft forever preferred_lft forever
```

Reply:
Per our discussion, this not relevant as we need the user to be able to specify the Postgres host IP.